UI/Panels/Settings/GeneralPanel.de.resx

```diff
@@ -164,18 +164,12 @@
   <data name="label8.Size" type="System.Drawing.Size, System.Drawing">
```

Comment:
Ideally the localization resource should not contain any size or location values. Why? Because if you move things around in the default language, it will reflect correctly in the localized version too.

I did that wrong in the past. Instead of positioning or re-sizing elements specfically for a language, I started to make labels just a bit wider in the default language and set auto-size property to false. 

Ideally you remove all `.Size`  and `.Location` properties from the `.de.resx`-file

Reply:
We can keep it the way it is this time, I will leave it up to you.